src/main/java/uk/gov/companieshouse/insolvency/data/model/InsolvencyDocument.java

```diff
@@ -20,17 +23,23 @@
 
     private final Updated updated;
 
+    @Version
+    private Long version;
+
+    private final String etag;
```

Comment:
There seems to be an existing etag in Companyinsolvency, shouldnt that be used instead?

Reply:
Will check further with @vchitti-kainos if we need to re-generate it in here

Reply:
Code change done.